clearwater-socket-factory/clearwater-socket-factory-mgmt-service

```diff
@@ -34,33 +34,9 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-MGMT_WHITELIST_DIR=/etc/clearwater-socket-factory/management.d
-SIGNALING_WHITELIST_DIR=/etc/clearwater-socket-factory/signaling.d
+WHITELIST_DIR=/etc/clearwater-socket-factory/management.d
+NAMESPACE=management
 
-get_daemon_args()
-{
-  . /etc/clearwater/config
+. /usr/share/clearwater/bin/clearwater-socket-factory-common
```

Comment:
The split between clearwater-socket-factory-common and this script feels a bit unclean. For example, you pass clearwater-socket-factory-common some parameter through undocumented environment variables. I think the code in clearwater-socket-factory-common would be better as functions that are called in the management and signaling-specific scripts.